app/javascript/chat/chat.jsx

```diff
@@ -1774,11 +1779,21 @@ export default class Chat extends Component {
         href='#/'
         data-content={dataContent}
       >
-        <img
-          src={ConfigImage}
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 490 490"
```

Comment:
@ludwiczakpawel Can I remove line 1784?

Reply:
I actually copied and pasted everything in our `info.svg` file into this file here.

Reply:
I think that info.svg is legacy icon from before crayons era :)

Reply:
I'll update the content of that file :)